devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -730,8 +730,8 @@ public void run() {
                     process.destroy();
                     try {
                         process.waitFor();
-                    } catch (InterruptedException ignored) {
-                        getLog().warn("Unable to properly wait for dev-mode end", ignored);
+                    } catch (InterruptedException e) {
+                        getLog().warn("Unable to properly wait for dev-mode end", e);
```

Comment:
You should rest the interruption flag. (it was not done before)

Reply:
This is a shutdown hook, so does it really matter to do so?